compiler/src/vm/mod.rs

```diff
@@ -499,17 +467,13 @@ impl Vm {
         }
     }
 
-    fn send_to_channel(
-        &mut self,
-        performing_fiber: Option<FiberId>,
-        channel_id: ChannelId,
-        packet: Packet,
-    ) {
+    fn send_to_channel(&mut self, performer: Performer, channel_id: ChannelId, packet: Packet) {
         let channel = match self.channels.get_mut(&channel_id) {
             Some(channel) => channel,
             None => {
                 // The channel was a nursery that died.
-                if let Some(fiber) = performing_fiber {
+                // TODO: also panic a nursery performer
```

Comment:
Should this be addressed before merging?

Reply:
Oh, that todo is already implemented by the code below. I just forgot to delete it.